plugin/storage/es/esRollover.py

```diff
@@ -112,8 +112,10 @@ def create_index(client, name):
     try:
         create.do_action()
     except curator.exceptions.FailedExecution as e:
-        if "index_already_exists_exception" not in str(e) and "resource_already_exists_exception" not in str(e):
+        if ("index_already_exists_exception" or "resource_already_exists_exception") in str(e):
             raise e
+        else:
+            print("Index {} already exists".format(name))
```

Comment:
Aside from a log entry, how is it functionally different from before, when the exception was simply ignored?

Reply:
This just logs that the index was not created because it already exists